src/pages/resultsView/enrichments/GeneBarPlot.tsx

```diff
@@ -154,37 +154,41 @@ export default class GeneBarPlot extends React.Component<IGeneBarPlotProps, {}>
 
     @computed get toolbar() {
         return (
-            <div style={{ zIndex: 10, position: "absolute", top: "10px", right: "10px" }}>
-                <div className={styles.ChartControls}>
-                    <strong>{ this._label || this.defaultOption.label}</strong>
-                    <DefaultTooltip
-                        trigger={['click']}
-                        destroyTooltipOnHide={false}
-                        visible={this.isGeneSelectionPopupVisible}
-                        onVisibleChange={(visible) => {
-                            this.isGeneSelectionPopupVisible = visible;
-                        }}
-                        overlay={
-                            <GenesSelection
-                                options={this.geneListOptions}
-                                selectedValue={this.geneQuery}
-                                onSelectedGenesChange={(value, genes, label) => {
-                                    this._geneQuery = value;
-                                    this.selectedGenes = genes;
-                                    this._label = label;
-                                    this.isGeneSelectionPopupVisible = false;
-                                }}
-                                defaultNumberOfGenes={DEFAULT_GENES_COUNT}
-                                containerType={this.props.containerType} />
-                        }
-                        placement="bottomLeft"
-                    >
-                        <div>
-                            <button className="btn btn-default btn-xs">
-                                Select genes
-                            </button>
-                        </div>
-                    </DefaultTooltip>
+            <div>
```

Comment:
@adamabeshouse @dippindots this might be a good example where we could use react fragment!  <> instead of <div> since div is superfluous.  @dippindots try it out!